generators/getMetricFunctions.js

```diff
@@ -102,11 +102,11 @@ const metricsList = [
     metric: 'nvt',
     sheetMetricName: 'nvt_ratio',
     description: [
-      'Returns NVT (Network-Value-to-Transactions-Ratio Daily Market Cap / Daily Transaction Volume)',
+      'Returns NVT (Network-Value-to-Transactions-Ratio Daily Market Cap / Daily Token Circulation)',
```

Comment:
Few grammar changes/recommendations from Brian:
```
Returns NVT (Network-Value-to-Transactions-Ratio Daily Market Cap / Daily Token Circulation)

Since Daily Transaction Volume gets rather noisy and easy to manipulate by transferring the same tokens through a couple of addresses repeatedly, it’s not an ideal measure of a network’s economic activity. That’s why we also offer another way to calculate NVT by using Daily Token Circulation. This method filters out excess transactions and provides a cleaner overview of a blockchain’s daily transaction throughput.
```

Reply:
I took them into account and fixed them, thanks